INSTALL.md

```diff
@@ -0,0 +1,194 @@
+# Installation Guide
+
+## Prerequisites 
```

Comment:
Just for my understanding when putting this into the User docs.  Are these pre-requisites we can reasonably assume this audience (which is very technical) knows how to do (e.g. Postgres SQL installation, IdP client setup), or do I need to provide steps or references for them?

Reply:
Whether or not they know how to do it is irrelevant to us. We are not supporting end users in their own infrastructure.